contrib/msoffice/src/com/sun/jna/platform/win32/COM/office/MSOfficeDemo.java

```diff
@@ -103,9 +104,10 @@ public void testMSWord() throws IOException {
             // save the document and prompt the user
             msWord.Save(false, wdPromptUser);
         } catch (COMException e) {
-            if (e.getExcepInfo() != null) {
-                System.out.println("bstrSource: " + e.getExcepInfo().bstrSource);
-                System.out.println("bstrDescription: " + e.getExcepInfo().bstrDescription);
+            if (e instanceof COMInvokeException) {
```

Comment:
This should just be a dedicated catch of `COMInvokeException` and there should be a catch for `COMException` after it that just gets ignored.